webpack.config.js

```diff
@@ -0,0 +1,915 @@
+  ]),
+  "nightly_firstrun": resolveBundles([
+    "css/firefox/nightly-firstrun.scss"
+  ]),
+  "nightly_whatsnew": resolveBundles([
+    "css/firefox/whatsnew/whatsnew-nightly.scss"
+  ]),
+  "oldIE": resolveBundles([
+    "css/sandstone/old-ie.scss",
+    "css/sandstone/oldIE.less"
+  ]),
+  "oldIE-pebbles": resolveBundles([
+    "css/pebbles/base/oldIE.scss"
+  ]),
+  "page_not_found": resolveBundles([
+    "css/base/page-not-found.scss"
+  ]),
+  "participation-reporting": resolveBundles([
+    "css/mozorg/participation-reporting.scss"
+  ]),
+  "partnerships-distribution": resolveBundles([
+    "css/mozorg/partnerships-distribution.scss"
+  ]),
+  "pebbles": resolveBundles([
+    "css/pebbles/global.scss",
+    "css/pebbles/protocol-footer.scss",
+    "css/pebbles/protocol-nav.scss"
+  ]),
+  "pebbles-basic": resolveBundles([
+    "css/pebbles/basic.scss"
+  ]),
+  "plugincheck": resolveBundles([
+    "css/plugincheck/plugincheck.scss",
+    "js/plugincheck/plugincheck.js"
+  ]),
+  "press": resolveBundles([
+    "css/press/press.scss"
+  ]),
+  "press_speaker_request": resolveBundles([
+    "js/libs/modernizr.custom.inputtypes.js",
+    "js/press/speaker-request.js"
+  ]),
+  "privacy-day-egg": resolveBundles([
+    "css/mozorg/internet-health/privacy-day-egg.scss",
+    "js/libs/konami-code.js",
+    "js/mozorg/internet-health/privacy-day-egg.js"
+  ]),
+  "privacy_email": resolveBundles([
+    "css/privacy/privacy-email.scss"
+  ]),
+  "privacy_firefox": resolveBundles([
+    "js/base/uitour-lib.js",
+    "js/privacy/privacy-firefox.js"
+  ]),
+  "privacy_protocol": resolveBundles([
+    "css/privacy/privacy-protocol.scss",
+    "js/base/mozilla-article.js",
+    "js/privacy/privacy-protocol.js",
+    "protocol/js/protocol-sidemenu.js"
+  ]),
+  "product_pocket": resolveBundles([
+    "css/firefox/pocket.scss",
+    "js/base/mozilla-fxa-product-button-init.js",
+    "js/base/mozilla-fxa-product-button.js",
+    "js/base/mozilla-lazy-load.js",
+    "js/firefox/pocket.js"
+  ]),
+  "protocol-core": resolveBundles([
+    "css/protocol/protocol.scss"
+  ]),
+  "protocol-oldIE": resolveBundles([
+    "css/protocol/oldIE.scss"
+  ]),
+  "responsive": resolveBundles([
+    "css/base/notification-banner.less",
+    "css/sandstone/protocol-footer.scss",
+    "css/sandstone/protocol-nav.scss",
+    "css/sandstone/sandstone-resp.less",
+    "css/tabzilla/tabzilla-static.less"
+  ]),
+  "safebrowser": resolveBundles([
+    "css/firefox/features/safebrowser.scss"
+  ]),
+  "security": resolveBundles([
+    "css/security/security.scss"
+  ]),
+  "security-bug-bounty-hall-of-fame": resolveBundles([
+    "css/security/hall-of-fame.scss"
+  ]),
+  "site": resolveBundles([
+    "js/base/core-datalayer-page-id.js",
+    "js/base/dnt-helper.js",
+    "js/base/mozilla-cookie-helper.js",
+    "js/base/site.js"
+  ]),
+  "stub-attribution": resolveBundles([
+    "js/base/stub-attribution-init.js",
+    "js/base/stub-attribution.js"
```

Comment:
The JS errors are occuring because of the order the files in each bundle. If we we're using JS modules for importing dependencies this wouldn't be an issue, but bedrock currently handles bundling via concatenation. To fix the errors, I think you're gonna have to avoid the temptation to reorder things?

Here `stub-attribution-init.js` needs to come after `stub-attribution.js`. This is the same for the rest of the JS bundles.